Sphinx/Guides/Actions.rst

```diff
@@ -170,8 +170,13 @@ Use an Addon/Bundle Action
 
         .. code-block::
 
+            #deprecated use the new Mail binding
```

Comment:
Let's put the new one on top!

Reply:
To do it through the GH web client, go to Files Changed, click the elipsis at the top right, select Edit File and make the change.

Reply:
Or change it when you close this to add the signoff! :smile: